core/storage/blog/gae_models_test.py

```diff
@@ -0,0 +1,266 @@
+# coding: utf-8
+#
```

Comment:
Why is this necessary?

Reply:
The encoding information is used by the python parser to interpret the file using the given encoding. All other test files also have it.